tests/functional/force_auth.js

```diff
@@ -9,40 +9,151 @@ define([
 ], function (registerSuite, TestHelpers, FunctionalHelpers) {
   var thenify = FunctionalHelpers.thenify;
 
-  var clearBrowserState = thenify(FunctionalHelpers.clearBrowserState);
   var click = FunctionalHelpers.click;
   var createUser = FunctionalHelpers.createUser;
   var fillOutForceAuth = FunctionalHelpers.fillOutForceAuth;
+  var fillOutSignUp = thenify(FunctionalHelpers.fillOutSignUp);
   var openForceAuth = FunctionalHelpers.openForceAuth;
+  var testElementDisabled = FunctionalHelpers.testElementDisabled;
   var testElementExists = FunctionalHelpers.testElementExists;
+  var testElementTextInclude = FunctionalHelpers.testElementTextInclude;
   var testElementValueEquals = FunctionalHelpers.testElementValueEquals;
   var type = FunctionalHelpers.type;
   var visibleByQSA = FunctionalHelpers.visibleByQSA;
 
+  function testAccountNoLongerExistsErrorShown() {
+    return this.parent
+      .then(testElementTextInclude('.error', 'no longer exists'));
+  }
+
   var PASSWORD = 'password';
   var email;
 
   registerSuite({
-    name: 'force_auth with an existing user',
+    name: 'force_auth',
 
     beforeEach: function () {
       email = TestHelpers.createEmail();
 
+      return FunctionalHelpers.clearBrowserState(this);
+    },
+
+    'with an invalid email': function () {
       return this.remote
-        .then(createUser(email, PASSWORD, { preVerified: true }))
-        .then(clearBrowserState(this));
+        .then(openForceAuth({
+          // TODO - this is a discrepancy and should go to the 400 page,
+          // but that's for another PR.
```

Comment:
I should file an issue, though I already have a fix for it. :D I just don't want to open yet another PR until this one has merged.